rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -186,14 +321,60 @@ components:
       maxLength: 80
       pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
       example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    MetadataInformationResponse:
```

Comment:
✏️ This is way more complicated than CDR Team needs right now. I would prefer keeping things extremely simple until we have a need to add complexity.

Reply:
This supports our model going forward. For now that list will be empty to start with and will contain one entry for `irm` after. CDR can simply rely on the transformationStatus of the whole request to reach a final state and then retrieve the URI from the `irm` entry.